nav/dataset/dataset_pds3_newhorizons_lorri.py

```
@@ -1,13 +1,58 @@
+
+        Parameters:
+            row: Dictionary containing PDS3 index table row data.
+
+        Returns:
+            The file specification string from the row.
+        """
+
+        filespec = cast(str, row['FILE_SPECIFICATION_NAME'])
+        if not filespec.endswith(('_sci.lbl', '_eng.lbl')):
+            raise ValueError(f'Bad Primary File Spec "{filespec}" - '
+                             'expected "_sci.lbl" or "_eng.lbl"')
```

Ruff (0.14.0)

Avoid specifying long messages outside the exception class

(TRY003)